### DISCUSSION TITLE
How to disable backpack visibility?

### DISCUSSION THREAD

**Discussion by Knewest** [BeansGalaxy/Beans-Backpacks-2#143] [Feedback] [answered]
I've tried very hard to answer this myself, but to no result. How can I disable my backpack being visible? I want to use a backpack without it clipping through my cape...

**Comment by BeansGalaxy** [comment]
Two things...

1. Currently you can't but I've just added a system that disables the back slot rendering when you're wearing an armor with a bulky model. I could add a config option that will always disables backpack rendering. It would have to be a setting for the server/singleplayer though but it's an easy feature to add.
2. Is it a cape from another mod that's clipping through your backpack? Vanilla capes become disabled when a backpack's equipped so I assume it's another mod unless this is a bug.

**Comment by BeansGalaxy** [comment, marked as answer]
In version 20.1-0.28-v2 you can disable it's rendering in `beansbackpacks-common.json5` by switching the setting `always_disables_back_slot` to `true`. This setting has to be true on the server for it to hide them since I don't like the idea of players accessing your backpack with out it visible to you. 

Also I added support for the cape to display with your backpack. It should've been hidden if you have the backpack equipped so if it's a mod that adds capes and it's not working correctly, lmk that mod and I'll check out how I could add compatibility ;)

**Comment by Knewest** [comment]
Hi, thanks for replying and apologies for my late response. The mod I used for the cape was [Wavey Capes by tr7zw](https://modrinth.com/mod/wavey-capes). Thanks so much for your prompt support and maintaining of the mod.

**Reply by BeansGalaxy** [reply]
Don't worry about the response because I'm pretty late too! It looks like Wavey Capes also has this issue with the elytra slot mod. I'll keep looking through their code but right off the bat it doesn't look super straight forward to stop their cape rendering. I'll keep looking tho and thanks for YOUR support!
